### PR TITLE
Search user interface, still no functionality.

### PR DESCRIPTION
**Pull request by TomB-0**
WIP
Implemented changes to Post page - created search ui.

### CODE REVIEW

File: Karma/Views/Posts/Index.cshtml [hunk 1]
```diff
@@ -10,15 +10,50 @@
...
+                <div class="defaultOption">All</div>
+                <ul>
+                    <li>All</li>
+                    <li>Recent</li>
+                    <li>Clothes</li>
+                    <li>Furniture</li>
+                    <li>Crafts</li>
+                    <li>Tools</li>
+                    <li>Books</li>
+                    <li>Other</li>
```
Comment (ArmintasP) [inline Karma/Views/Posts/Index.cshtml:48]: This should be iterated using foreach, where elements are ItemTypes.

**Review by ArmintasP — COMMENTED** [review]
I agree with @baikauskaite , we could always focus on design later on.

File: Karma/Views/Posts/Index.cshtml [hunk 2]
```diff
@@ -10,15 +10,50 @@
     ViewData["Title"] = "All posts";
 }
 
-<h1>@ViewBag.Header</h1>
+<h1>
+    @ViewBag.Header
```
Comment (PauliusLu) [inline Karma/Views/Posts/Index.cshtml:14]: I think here should be end tag `</h1>`

File: Karma/Views/Posts/Index.cshtml [hunk 3]
```diff
@@ -10,15 +10,50 @@
...
-    <a asp-action="CreateRequest">Create Request</a>
-</p>
+            $(".dropdown ul li").click(function () {
+                var text = $(this).text();
+                $(".defaultOption").text(text);
+                $(".dropdown ul").removeClass("active");
+            });
+        });
+</script>
+</h1>
```
Comment (PauliusLu) [inline Karma/Views/Posts/Index.cshtml:32]: Is there a reason why `meta`, `link`, `script` tags are inside of h1?

File: Karma/Views/Posts/Index.cshtml (hunk 1 above)
Comment (PauliusLu) [reply Karma/Views/Posts/Index.cshtml:48]: For now it might be so, but definitely a change will be needed in near future or sooner

**Pull request closed without merging**